atlasdb-timelock-server/src/integTest/java/com/palantir/atlasdb/timelock/TimeLockServerIntegrationTest.java

```diff
@@ -52,6 +53,8 @@
 import io.dropwizard.testing.ResourceHelpers;
 import io.dropwizard.testing.junit.DropwizardAppRule;
 
+@Ignore("Observed ConcurrentModificationException-related flakes (e.g. build #611 on internal CircleCI)."
```

Comment:
We're already moving away from atomix (due to correctness issues with it) so these will be deleted soon